Unit test case for basicget.js

tanaysingh3484 opened the pull request:
Created a Unit-test file for the `basicget.js` file, using the `Chai` assertion library.

chughts commented on Node.js/test/basicget.test.js:
```diff
@@ -0,0 +1,234 @@
+const { buildMQDetails, ccdtCheck, initialise, connx, open, close, disconnect, getMessage } = require('../basicget');
```
README needs updating with instructions on how to run the tests.

chughts replied:
package.json will need updating to allow for `npm test` command

chughts commented on Node.js/test/basicget.test.js:
```diff
@@ -0,0 +1,234 @@
+
+    it('Should perform MQGET', async () => {
+        for (let i = 0; i < envConfigLength; i++) {
+            await buildMQDetails(MQDetails, credentials, i);
```
This is a copy and paste of the put logic. Instead, just run `basicput`, then test that the get functions run.

tanaysingh3484 commented:
- Updated the `MQGET` test by adding an `exec` statement to first execute the `basicput.js` file, following which the `getMessage` function is called
- Added a `README` for the test directory, which contains the information needed to run these testcases
- Updated the `basicget.js` with the required `module.exports`.

chughts reviewed (approved):
Seems odd to export the functions, just so they can be tested, but I guess this is a result of having a single module node.js app.
